### ISSUE TITLE
Casper two not supported

### ISSUE THREAD

**Issue by patarol** [rstudio/blogdown#322]
I'm trying to render the `casper two` theme and it seems that `blogdown` can't render it.

Here's a screenshot:
![screen shot 2018-08-06 at 23 45 14](https://user-images.githubusercontent.com/31883497/43742596-d3873532-99d2-11e8-991e-d4c9f0159530.png)

Is that possible to accomplish two render this theme using `blowdown`?

Thank you.

**Comment by yihui** [comment]
There might be error messages in your R console. Please try the preview version of RStudio if you are using RStudio: https://www.rstudio.com/products/rstudio/download/preview/

Related: #257

Once you fix the Hugo error, you should be able to preview your website.

**Comment by patarol** [comment]
Thank you @yihui! It worked!!

However I can't manage to render `highcharter` with `Rmarkdown`.
I've tried [this](https://stackoverflow.com/questions/30509866/for-loop-over-dygraph-does-not-work-in-r) approach and nothing happens...
Here's what I get:

![error_highcharter](https://user-images.githubusercontent.com/31883497/43759432-859da3a4-9a1f-11e8-929a-2af12aafe135.png)

Maybe here isn't the place for this issue since I've started with a render site issue.

Thank you!
